README.org

```diff
@@ -121,44 +121,44 @@
    ~lsp-clients.el~ while others require installing additional packages which provide
    server specific functionality.
 
-   | Language              | Language Server                           | Built-in       | Installation command                                                                                                 | Debugger                     |
```

Comment:
The formatting changes are caused by the fact that one is performing the formatting with links display enabled and the other is performing the alignment with link display disabled(use `org-toggle-link-display`). Can you do the formatting so we will have only the required changes?

Reply:
Ah, nice catch. Fixed!